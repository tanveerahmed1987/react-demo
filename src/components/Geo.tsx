import React from "react";
import { TwoColumn } from "./styles";
import { Continent } from "./Country/Continent";
import { Countries } from "./Country/Countries";

const Geo: React.FC = () => {
  return (
    <TwoColumn>
      <Continent />
      <Countries />
    </TwoColumn>
  );
};

export default Geo;
