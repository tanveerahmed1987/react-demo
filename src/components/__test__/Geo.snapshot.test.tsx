// components/__tests__/Counter.snapshot.test.tsx
import * as React from "react";
import Geo from "../Geo";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer.create(<Geo />).toJSON();
  expect(tree).toMatchSnapshot();
});
