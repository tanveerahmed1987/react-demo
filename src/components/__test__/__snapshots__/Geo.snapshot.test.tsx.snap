// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-jRQAMF kyLWzu"
>
  <div>
    <div
      className="sc-gsDJrp grQNCr"
    >
      <input
        className="sc-bdvvaa jRdPwq"
        onChange={[Function]}
        placeholder="Enter continent code..."
        type="text"
      />
    </div>
  </div>
  <div>
    <div
      className="sc-gsDJrp grQNCr"
    >
      <input
        className="sc-bdvvaa jRdPwq"
        onChange={[Function]}
        placeholder="Enter country code..."
        type="text"
      />
    </div>
  </div>
</div>
`;
